docs/src/misc.rst

```diff
@@ -273,6 +273,22 @@ API
 
     .. versionadded:: 1.6.0
 
+.. c:function:: int uv_os_tmpdir(char* buffer, size_t* size)
+
+    Gets the temp directory. On Windows, `uv_os_tmpdir()` uses `GetTempPathW()`.
+    On all other operating systems, `uv_os_tmpdir()` uses the first environment
+    variable found in the ordered list `TMPDIR`, `TMP`, `TEMP`, and `TEMPDIR`.
+    If none of these are found, the path `"/tmp"` is used, or, on Android,
+    `"/data/local/tmp"` is used. The temp directory is stored in `buffer`. When
+    `uv_os_tmpdir()` is called, `size` indicates the maximum size of `buffer`.
+    On success or `UV_ENOBUFS` failure, `size` is set to the string length of
+    `buffer`.
```

Comment:
`(which doesn't include the terminating null)`

I think its worth making very explicit that string length is what `strlen()` would return.